code/cryptography/src/autokey_cipher/autokey_cipher.java

```diff
@@ -0,0 +1,107 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashMap;
+
+public class autokey
+{
+	public static String createkey(String plaintext, int keycount, HashMap<Character, Integer> hashmap1, HashMap<Integer, Character> hashmap2)
```

Comment:
Fix indentation in this file to 4 spaces.

Reply:
indentation is used clang-format(x64)

Reply:
@ravireddy07 That was only for C/C++. For Java we use 4 space indentation.

Reply:
Code indented. please check now. @arnavb

Reply:
Indented the code on github only. 4-space Indenattion.